watchtower-plugin/src/net/http.rs

```diff
@@ -154,6 +155,31 @@ pub async fn send_appointment(
     }
 }
 
+/// Handles a ping request to the tower.
+pub async fn ping(server_url: &str) -> Result<(), String> {
+    let client = Client::new();
+    let response = client
+        .get(&format!("{}/{}", server_url, Endpoint::Ping))
+        .send()
+        .await
+        .map_err(|err| format!("Failed to send ping request: {}", err))?;
+
+    if response.status().is_success() {
+        debug!("Ping successful!");
+        Ok(())
+    } else {
+        let status_code = response.status();
+        let error_msg = response
```

Comment:
nit: You don't need to define intermediate variables for this.

Reply:
sure!!